templates/navigation.hbs

```diff
@@ -1,6 +1,9 @@
 {{! Maintainers - Daryl Hedley}}
 {{! All links in the nav bar should have an attritbue of data-event, navigationView uses this to fire a global event}}
 <div class="navigation-inner clearfix" role="navigation" aria-label="{{_globals._accessibility._ariaLabels.navigation}}">
+    {{#if _accessibility._isSkipNavigationEnabled}}
+        <button class="skip-nav-link" aria-label="Skip navigation" data-event="skipNavigation" role="button">Skip Navigation</button>
```

Comment:
shouldn't the aria-label be in course.json to allow it to be changed/translated?